variables.tf

```diff
@@ -15,6 +15,13 @@ variable "nomad_host_volume" {
   default     = "persistence"
 }
 
+variable "consul_tags" {
+  type        = list(string)
+  description = "List of tags for services announces in consul service catalog"
+  default     = [" "]
```

Comment:
Think default should be empyt or prehaps `"minio"`. WDYT?

Reply:
I´ll leave it empty